Philips.CodeAnalysis.MaintainabilityAnalyzers/Readability/SplitMultiLineConditionOnLogicalOperatorAnalyzer.cs

```diff
@@ -144,9 +144,11 @@ private bool IsIllegalLineBreakToken(SyntaxToken token)
 			// We allow line breaks on the following tokens:
 			// - ( for more complex logic expressions
 			// - && and || as these are the logical operators.
+			// => for lambdas
 			return !token.IsKind(SyntaxKind.OpenParenToken) &&
 			       !token.IsKind(SyntaxKind.AmpersandAmpersandToken) &&
-			       !token.IsKind(SyntaxKind.BarBarToken);
+			       !token.IsKind(SyntaxKind.BarBarToken) &&
+			       !token.IsKind(SyntaxKind.EqualsGreaterThanToken);
```

Comment:
Are you sneaking in functionality? ;-)

Reply:
Actually, the PR title should probably be "refactor:" rather than "fix:" right?

Reply:
I sneaked in a false positive fix. Can make a separate Issue and PR if you prefer